### PR TITLE
Update alune to set 13

### PR DESCRIPTION
**Pull request by catasdd**

### CODE REVIEW

**Review by Kyrluckechuck — APPROVED** [review]
Makes enough sense assuming you tested the bot for the gameplay loop for the new season -- things like the new popups and such can cause issues if they change a lot between seasons

File: alune/images.py [hunk 1]
```diff
@@ -165,31 +165,20 @@ def get_default_traits(cls):
...
+    EMISSARY = auto()
+    ENFORCER = auto()
+    EXPERIMENT = auto()
+    FAMILY = auto()
+    FIRELIGHT = auto()
+    REBEL = auto()
+    SCRAP = auto()
 
 
 class DawnOfHeroesTrait(StrEnum):
```
Comment (akshualy) [inline alune/images.py:184]: It would make sense to change this as well if there's a new second game mode

**Review by akshualy — CHANGES_REQUESTED** [review]
I just had the chance to check and there is no new second gamemode. I'm not sure if you understood, but dawn of heroes was not the normal TFT. Let's merge the traits into the normal traits class and leave the dawn of heroes code alone.

**Comment by catasdd** [comment]
> I just had the chance to check and there is no new second gamemode. I'm not sure if you understood, but dawn of heroes was not the normal TFT. Let's merge the traits into the normal traits class and leave the dawn of heroes code alone.

I think I did it, but check it please.

**Comment by akshualy** [comment]
> > I just had the chance to check and there is no new second gamemode. I'm not sure if you understood, but dawn of heroes was not the normal TFT. Let's merge the traits into the normal traits class and leave the dawn of heroes code alone.
> 
> I think I did it, but check it please.

looks better, but you still deleted the dawn of heroes stuff - i think that'll cause an error. running workflow to make sure

**Comment by catasdd** [comment]
> > > I just had the chance to check and there is no new second gamemode. I'm not sure if you understood, but dawn of heroes was not the normal TFT. Let's merge the traits into the normal traits class and leave the dawn of heroes code alone.
> > 
> > 
> > I think I did it, but check it please.
> 
> looks better, but you still deleted the dawn of heroes stuff - i think that'll cause an error. running workflow to make sure
> 
> edit: yea as i thought, you're referring to an enum member that doesn't exist; you can replace [cls.DAWNBRINGER] with [] honestly

should I add dawnbringer stuff again or leave it like this?

**Review by akshualy — APPROVED** [review]

**Comment by akshualy** [comment]
lgtm

**Review by Kyrluckechuck — COMMENTED** [review]
Could you run a `git checkout HEAD -- alune/images/traits/dawn_of_heroes/` to restore all of the dawn of heroes stuff since that isn't really related to the aim of the PR?

We may want to clean those up as well, but can do that separately

**Comment by subzdelta** [comment]
Would like to know if someone can send the exe for the set 13 as I've ran some problems trying to merge it myself, thanks in advance!

**Comment by enbge** [comment]
pls update ty

**Pull request merged**